redux/auth/thunks.ts

```
@@ -34,6 +41,37 @@ export const signIn = createAsyncThunk(
+        thunkAPI.dispatch(
+          setSnackbar({
+            msg: `${newKey} ${string.join(' ')}`,
+            type: 'error',
+            setTime: 3000,
+          }),
+        );
+        throw data.err;
+      }
+    } catch (err) {
+      throw err;
+    }
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)